test/e2e/infra/vagrant/push_antrea.sh

```diff
@@ -3,12 +3,16 @@
 function usage() {
     echo "Usage: push_antrea.sh [--prometheus] [-fc|--flow-collector <Address>] [-fa|--flow-aggregator] [-h|--help]
     Push the latest Antrea image to all vagrant nodes and restart the Antrea daemons
-          --prometheus                Deploy Prometheus service to scrape metrics from Antrea Agents and Controllers
-          --flow-collector <Address>  Provide the IPFIX flow collector address to collect the flows from the Flow Aggregator service
-                                      It should be given in the format IP:port:proto. Example: 192.168.1.100:4739:udp
-                                      Please note that with this option we deploy the Flow Aggregator Service along with Antrea.
-          --flow-aggregator           Deploy Flow Aggregator along with Antrea.
-                                      It is automatically deployed if --flow-collector is used."
+          --prometheus                 Deploy Prometheus service to scrape metrics
+                                       from Antrea Agents and Controllers.
+          --flow-collector <Addr|ELK>  Provide either the external IPFIX collector
+                                       address or specify 'ELK' to deploy the ELK
+                                       flow collector. The address should be given
+                                       in the format IP:port:proto. Example: 192.168.1.100:4739:udp.
+                                       Please note that with this option we deploy the Flow Aggregator Service.
+          --flow-aggregator            Deploy Flow Aggregator along with Antrea.
+                                       This is to run Flow Aggregator e2e tests.
```

Comment:
The description is not very clear to me, maybe we can change to something like `Upload Flow Aggregator manifests and docker images to prepare for running Flow Aggregator e2e tests` since it does not deploy the flow aggregator.

Reply:
Agreed. This has to be edited after the modification.